xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4629,6 +4641,13 @@ void CVideoPlayer::GetVideoStreamInfo(int streamId, SPlayerVideoStreamInfo &info
   if (s.name.length() > 0)
     info.name = s.name;
 
+  if (s.width == 0 || s.height == 0)
```

Comment:
This is some left over and I am going to drop it because it did not change anything. Some parameters like aspect ratio or audio channels get updated later by stream players.

> checking against string length of info.videoCodecName is acceptable

codec name is always valid and does never change.